validator/src/cli.rs

```diff
@@ -2325,6 +2335,7 @@ impl DefaultArgs {
                 .retry_pool_max_size
                 .to_string(),
             rpc_threads: num_cpus::get().to_string(),
```

Comment:
this needs to be set a lot lower

Reply:
I'll tweak these and see how the choice of cpus affects the benchmark latencies.